scripts/config-list.js

```diff
@@ -58,6 +60,24 @@ mc.getAsync('limits')
       else {
         output.allowedIPs = data
       }
+      return mc.getAsync('allowedEmailDomains')
+    }
+  )
+  .then(
+    function (data) {
+      if (!data) { console.error('no allowedEmailDomains set') }
```

Comment:
Haven't seen if statements like this in any other project, but it is consistent with this file :)